hbp_knowledge/proteostasis/morimoto2015.bel

```diff
@@ -75,333 +75,440 @@ SET Citation = {"PubMed", "25784053"}
 cochaperones, decline with protein aggregation (64, 65)."
+SET Confidence = "High"
 path(MESH:"Protein Aggregation, Pathological") negativeCorrelation p(HGNC:HSPA1A)
 path(MESH:"Protein Aggregation, Pathological") negativeCorrelation p(HGNC:HSPA1B)
 path(MESH:"Protein Aggregation, Pathological") negativeCorrelation p(HGNC:DNAJB1)
+UNSET Confidence
 
 SET Evidence = "Experiments examining the effects of Aβ on
 proteasomal activity in vitro revealed an inhibitory effect on the chymotrypsin-like
 properties of the 20S core (73), consistent with observations of impaired proteasome
 function in AD patient brains (74). "
+SET Confidence = "High"
 a(CHEBI:"amyloid-beta") -| act(p(FPLX:Proteasome))
 SET MeSHAnatomy = "Brain"
 path(MESH:"Alzheimer Disease") negativeCorrelation act(p(FPLX:Proteasome))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = " Mice deficient for the autophagy-related genes
 Atg5 and Atg7 exhibit severe neurodegeneration (84, 85), and the expression of disease-
 associated proteins is reported to exert differential inhibitory effects on autophagic
 pathways."
+SET Confidence = "High"
 SET Species = "10090"
 p(MGI:Atg5) -| bp(HP:Neurodegeneration)
 p(MGI:Atg7) -| bp(HP:Neurodegeneration)
 UNSET Species
+UNSET Confidence
 
 SET Evidence = " Consistent with a role of autophagy in disease, AD patient tissues exhibit
 impaired initiation of macroautophagy and an excess of autophagic vacuoles in dystrophic
 neurites, possibly due to impaired targeting of the vacuolar ATPase to the lysosome (86, 87)."
+SET Confidence = "High"
 path(MESH:"Alzheimer Disease") negativeCorrelation bp(GO:macroautophagy)
+UNSET Confidence
 
 SET Evidence = "In contrast, α-synuclein overexpression impairs autophagy in mammalian cells and mice
 through reduced expression of RAB1A, thereby inhibiting autophagosome formation (88). "
+SET Confidence = "Medium"
 p(HGNC:SNCA) -| bp(GO:autophagy)
+SET Species = "10090"
 p(HGNC:SNCA) -| p(HGNC:RAB1A)
 p(HGNC:RAB1A) -> bp(GO:"autophagosome assembly")
+UNSET Species
+UNSET Confidence
 
 SET Evidence =  "In contrast to findings in HD, AD, and PD, a recent study has suggested that autophagy
 is enhanced in ALS mice."
+SET Confidence = "High"
 SET Species = "10090"
 path(MESH:"Amyotrophic Lateral Sclerosis") -> bp(GO:autophagy)
 UNSET Species
+UNSET Confidence
 
+#evidence for the commeneted BEL statements?
 SET Evidence = " Reduced levels of lipofuscin, LC3, and p62 have been observed in
 motor neurons of SOD1(G85R) mice (92). Treatment with the autophagy inhibitor chloroquine
 restored lipofuscin, LC3, and p62 levels in motor neurons, suggesting that mutant SOD1
 causes hyperactive autophagy in mice (92). "
 #p62 is SQSTM1
+SET Confidence = "Low"
 SET Species = "10090"
 SET Cell = "motor neuron"
-p(HGNC:SOD1, var("p.Gly576Arg")) -| a(CHEBI:"N-retinylidene-N-retinylethanolamine")
+#p(HGNC:SOD1, var("p.Gly576Arg")) -| a(CHEBI:"N-retinylidene-N-retinylethanolamine")
```

Comment:
lipofuscin is the same as N-retinylidene-N-retinylethanolamine, forgot to add this as a comment...